ecospheres_migrator/transformations/change-language.xsl

```diff
@@ -7,7 +7,7 @@
   <xsl:strip-space elements="*"/>
 
   <xsl:template match="/gmd:MD_Metadata/gmd:language/gco:CharacterString/text()">
-    <xsl:text>xxx</xsl:text>
+    <xsl:text>eng</xsl:text>
```

Comment:
So that is works for real when using a GN for tests.

Reply:
I should have put klingon